code/game/objects/structures/ai_core.dm

```diff
@@ -189,7 +189,10 @@
 						return
 
 					var/mob/living/brain/B = M.brainmob
+					/* honk start -- Remove checking for allow_ai flag (makes buildable AIs regardless of ALLOW_AI config)
 					if(!CONFIG_GET(flag/allow_ai) || (is_banned_from(B.ckey, "AI") && !QDELETED(src) && !QDELETED(user) && !QDELETED(M) && !QDELETED(user) && Adjacent(user)))

```

Comment:
Instead of commenting out the line, and creating a new version. Change the original line, and mark it with a `// honk --` comment.

Reply:
Done